unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/HUD/Minimap/MinimapHUDController.cs

```diff
@@ -102,6 +124,21 @@ public void ReportScene()
         WebInterface.SendReportScene($"{coords.x},{coords.y}");
     }
 
+    public void SetHomeScene(bool isOn)
+    {
+        var coords = playerCoords.Get();
+        if (playerCoords == homeCoords)
+        {
+            if (!isOn)
+                WebInterface.SetHomeScene($"0,0");
+        }
+        else
+        { 
+            if(isOn)
+                WebInterface.SetHomeScene($"{coords.x},{coords.y}");
```

Comment:
please instead of referencing `WebInterface` directly, we should inject an interface that sets the home scene, so follow the dependency inversion principle